README.md

```diff
@@ -84,14 +96,33 @@ or
 
 ### TypeScript
 
-In order to use this config in TypeScript project make sure you have installed following dependencies:
+Extend your ESLint config using `plugin:import/typescript`, for example:
 
-* [`@typescript-eslint/eslint-plugin`](https://yarnpkg.com/en/package/@typescript-eslint/eslint-plugin)
-* [`@typescript-eslint/parser`](https://yarnpkg.com/en/package/@typescript-eslint/parser)
-* [`typescript`](https://yarnpkg.com/en/package/typescript)
+```json
+{
+  "extends": [
+    "@callstack",
+    "plugin:import/typescript"
```

Comment:
We should add this to the plugin by default. We can't extend in an `override`, so need to add these manually:

```js
settings: {
  'import/extensions': ['.js', '.ts', '.tsx'],
  'import/parsers': {
    '@typescript-eslint/parser': ['.ts', '.tsx'],
  },
  'import/resolver': {
    node: {
      extensions: ['.js', '.ts', '.tsx'],
    },
  },
},
```

Wdyt @thymikee ?